### PR TITLE
INT-40 submit review

### PR DESCRIPTION
**Pull request by rkaczka-vd**

### CODE REVIEW

File: IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs [hunk 1]
```diff
@@ -57,5 +57,13 @@ public interface ISubmissionsClient
         /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asyncronous operations.</param>
         /// <returns><c><see cref="ISubmission"/></c> with provided id.</returns>
         Task<ISubmission> GetAsync(int submissionId, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Generates <seealso cref="ISubmission"/>'s result
+        /// </summary>
+        /// <param name="submissionId"></param>
+        /// <param name="cancellationToken"></param>
```
Comment (szymon-socha) [inline IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs:65]: Missing parameters comments.

File: IndicoV2.IntegrationTests/Reviews/ReviewsClientTests.cs [hunk 2]
```diff
@@ -14,46 +17,59 @@ public class ReviewsClientTests
...
+                var jobResult = await _jobAwaiter.WaitReadyAsync(jobId, TimeSpan.FromSeconds(1));
+                
+                // Assert
+                jobResult.Should().NotBeNullOrEmpty();
+            }
+            catch (JobNotSuccessfulException)
+            {
+                // TODO: fix the review
+                // Jobs's errorr: "Extraction predictions must be a list"
+                Assert.Fail("TODO: investigate what's wrong");
```
Comment (szymon-socha) [inline IndicoV2.IntegrationTests/Reviews/ReviewsClientTests.cs:69]: I guess that we're leaving that for now, right?

**Review by szymon-socha — APPROVED** [review]

**Review by goatrocks — APPROVED** [review]

File: IndicoV2.IntegrationTests/Reviews/ReviewsClientTests.cs (hunk 2 above)
Comment (rkaczka-vd) [reply IndicoV2.IntegrationTests/Reviews/ReviewsClientTests.cs:69]: I'm waiting on clarification of how to make this work.

File: IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs (hunk 1 above)
Comment (rkaczka-vd) [reply IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs]: fixed

**Pull request merged**